go.mod

```
@@ -0,0 +1,5 @@
+module github.com/135yshr/go_gotest
```

tddbc orgなのでこっちが正しそうです

```suggestion
module github.com/tddbc/go_gotest
```

---

あ。。確かに。ありがとうございます

---

修正しました